tox.ini

```diff
@@ -20,6 +20,7 @@ deps =
     transaction
     zc.lockfile
     zdaemon
+    zodbpickle
```

Comment:
The deps block is proceeded by this comment, so I assume somebody had a reason for listing all the deps; I just followed along:

```
# without explicit deps, setup.py test will download a bunch of eggs into $PWD
deps =
    BTrees
    ...
```